router/src/validation.rs

```diff
@@ -266,10 +271,19 @@ impl Validation {
             .unwrap_or(Ok(None))?;
 
         // Validate inputs
-        let (inputs, input_length, max_new_tokens) = self
+        let (inputs, _input_length, max_new_tokens) = self
             .validate_input(request.inputs, truncate, max_new_tokens)
             .await?;
 
+        let (negative_inputs, input_length, max_new_tokens) = self
+            .validate_input(negative_inputs, truncate, Some(max_new_tokens))
+            .await?;
+
+        let logit_biases: Vec<LogitBias> = logit_bias
```

Comment:
`logit_biases` makes sense as a name for `Vec<LogitBias>`, however this is immediately renamed to `logit_bias` when passed to `NextTokenChooserParameters`. 

It might make sense to use the name `logit_bias` instead, OR rename all of the instances of `Vec<LogitBias>` to `logit_biases`.

```suggestion
        let logit_bias: Vec<LogitBias> = logit_bias
```